app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        vaccine = visitor["vaccine"]
+        expiration_date = vaccine["expiration_date"]
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(expiration_date)
+
+        if not visitor.get("wearing_a_mask", False):
```

Comment:
visitor.get() can return None 
not None it is True
default parameter False is redundant here